wolox-books/src/app/screens/register/components/register-form/register-form.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-register-form',
+  templateUrl: './register-form.component.html',
+  styleUrls: ['./register-form.component.scss']
+})
+export class RegisterFormComponent implements OnInit {
+
+  registerForm: FormGroup;
+
+  constructor() { }
+
+  ngOnInit() {
+    this.registerForm = new FormGroup({
```

Comment:
i think you can initialize the property when you declare it, no need for doing it in the ngOnInit method